src/components/ApplyFiltersModal/DefaultFilter.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import Checkbox from "@leafygreen-ui/checkbox";
+import Tooltip from "@leafygreen-ui/tooltip";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+import { ParsleyFilter } from "gql/generated/types";
+import { Filter } from "types/logs";
+
+interface DefaultFilterProps {
+  filter: ParsleyFilter;
+  urlFilters: Filter[];
+}
+
+const DefaultFilter: React.FC<DefaultFilterProps> = ({
+  filter,
+  urlFilters,
+}) => {
+  const alreadyActive = !!urlFilters.find((f) => f.name === filter.expression);
+
+  return (
+    <DefaultFilterContainer data-cy="default-filter">
+      <Checkbox
+        checked={alreadyActive}
+        disabled={alreadyActive}
+        label={
+          <>
+            {filter.expression}
+            {alreadyActive && (
+              <Tooltip
+                data-cy="default-filter-tooltip"
+                trigger={
+                  <IconContainer>
+                    <Icon glyph="InfoWithCircle" />
+                  </IconContainer>
+                }
+                triggerEvent="hover"
+              >
+                This filter is already active.
+              </Tooltip>
+            )}
+          </>
+        }
+        name={filter.expression}
+      />
+      <BadgeContainer>
```

Comment:
Sorry if I missed the designs but have you considered reusing the existing filter components? It would be nice to have some consistency between them and this component. 


<img width="1169" alt="image" src="https://github.com/evergreen-ci/parsley/assets/4605522/9729a5ff-146c-4728-a603-6cfed2bd6ffe">

Reply:
I'm not entirely sure about this pattern as I feel like it's somewhat of a waste of space to show something that will never be interactive for any of these filters. TBH I'm not even sure I want to show the case/match values in the first place, because I feel like most users will be like “oh yay, helpful filters someone else defined for me” and ignore everything but the string value anyways (and they can always just check the properties manually in the side panel). But I also think this would be super easy to change in the future with little detriment to user experience =]

Reply:
good point, we can also change the design in that ticket if needed!!!